OpenConsole.sln

```diff
@@ -3375,6 +3379,118 @@ Global
 		{8222900C-8B6C-452A-91AC-BE95DB04B95F}.Release|x64.Build.0 = Release|x64
 		{8222900C-8B6C-452A-91AC-BE95DB04B95F}.Release|x86.ActiveCfg = Release|Win32
 		{8222900C-8B6C-452A-91AC-BE95DB04B95F}.Release|x86.Build.0 = Release|Win32
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|Any CPU.ActiveCfg = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|Any CPU.Build.0 = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|ARM.ActiveCfg = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|ARM.Build.0 = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|ARM64.ActiveCfg = AuditMode|ARM64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|ARM64.Build.0 = AuditMode|ARM64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|DotNet_x64Test.ActiveCfg = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|DotNet_x64Test.Build.0 = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|DotNet_x86Test.ActiveCfg = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|DotNet_x86Test.Build.0 = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|x64.ActiveCfg = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|x64.Build.0 = AuditMode|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|x86.ActiveCfg = AuditMode|Win32
+		{06EC74CB-9A12-428C-B551-8537EC964726}.AuditMode|x86.Build.0 = AuditMode|Win32
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|Any CPU.ActiveCfg = Debug|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|Any CPU.Build.0 = Debug|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|ARM.ActiveCfg = Debug|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|ARM.Build.0 = Debug|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|ARM64.ActiveCfg = Debug|ARM64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|ARM64.Build.0 = Debug|ARM64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|DotNet_x64Test.ActiveCfg = Debug|x64
+		{06EC74CB-9A12-428C-B551-8537EC964726}.Debug|DotNet_x64Test.Build.0 = Debug|x64

```

Comment:
FYI your new projects have been selected to build in the DotNet platforms (not necessary) and the Fuzzing configuration (also not necessary)

Best  way to fix is to delete all of the `.Build.0` lines for these projects + `DotNet_*` and `Fuzzing|*`